package.json

```diff
@@ -31,5 +31,8 @@
     "solhint": "^3.3.6",
     "solhint-config-keep": "github:keep-network/solhint-config-keep",
     "typescript": "^4.3.2"
+  },
+  "dependencies": {
+    "@thesis/solidity-contracts": "github:thesis/solidity-contracts#682e515f6d21da5a1d29956f53ba47f8294ba1bb"
```

Comment:
This will have to be changed into NPM package reference. Keeping it like that until we have an agreement on using `ERC20WithPermit`.

Reply:
Actually... it does not have to. We can keep referencing version from GitHub 🤷 

Updated to the most recent version in https://github.com/threshold-network/solidity-contracts/pull/3/commits/579d082bc04733c0095512930ee9d3b6a4432fe2.